webapp/src/main/resources/db/migration/V71__Add_scheduled_jobs.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE scheduled_job (
+    id CHAR(36) PRIMARY KEY NOT NULL,
+    repository_id BIGINT NOT NULL,
+    job_type SMALLINT DEFAULT NULL,
+    cron VARCHAR(50) DEFAULT NULL,
+    properties LONGTEXT NOT NULL,
+    job_status SMALLINT NOT NULL,
+    start_date TIMESTAMP NULL DEFAULT NULL,
+    end_date TIMESTAMP NULL DEFAULT NULL,
+    enabled TINYINT DEFAULT 1
```

Comment:
Same comment as for the entity re the data type for a bool, preferable to use the BIT data type

Reply:
Fixed @garionpin